### ISSUE TITLE
Some uncaught exception

### ISSUE THREAD

**Issue by HMAZonderland** [wakatime/WakaTime.novaextension#13]
Extension log told me to report an issue, so here we are:

```
[WakaTime] [DEBUG] Sending heartbeat:
<home_directory>/.wakatime/wakatime-cli/wakatime-cli --file <remote file> --plugin "\"nova/9.1 nova-wakatime/2.0.3"" --language php --local-file /tmp/1647946541055
```

and this response:
```
Extension encountered an uncaught exception:
<home_directory>/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 397, Column 16)
Error: Could not find an executable for process: <home_directory>/.wakatime/wakatime-cli/wakatime-cli
    @[native code]
    sendHeartbeat@file://<home_directory>/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:397:16
    onEvent@file://<home_directory>/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:320:18

WakaTime issues may be reported at https://github.com/wakatime/WakaTime.novaextension/issues
```

**Comment by HMAZonderland** [comment]
I see in my logging, after restarting Nova, it downloads a cli - zip from AWS? This does lag my editor. Is this normal?

```
WakaTime[12:06:56.687000] [WakaTime] [DEBUG] Initializing version 2.0.3
WakaTime[12:06:56.688000] [WakaTime] [DEBUG] Downloading wakatime-cli from https://wakatime-cli.s3-us-west-2.amazonaws.com/mac-x86-64/wakatime-cli.zip
```

I have the wakatime-cli running:
```
% which wakatime-cli
/opt/homebrew/bin/wakatime-cli
```

It wants a directory, but cannot find it:
```
[WakaTime] [ERROR] Failed to extract wakatime-cli.zip with error: checkdir error:  
<home_directory>/.wakatime/wakatime-cli exists but is not directory
````

and sprays the extension log with (related) errors:
```
checkdir error:  <home_directory>/.wakatime/wakatime-cli exists but is not directory

                 unable to process wakatime-cli/certifi/.
```

**Comment by gandarez** [comment]
Closed by #14
